{"uid":"90f8f0a12e35c595","name":"Create a New Order for store","fullName":"TestOrder.TestOrder#test_create_order_store_1___frugal_latido_com_np____Test_123____Ek__male_____Black_Sheep____38____Silver____Black____No____40____Fit___8__7___33___11__5__9__12__46___6___5_6___Test_002___20000__1000___101011__","historyId":"fcd5b12ae79643d7e5524826b092c938","time":{"start":1687502442891,"stop":1687502499133,"duration":56242},"description":"Create a New Order for store","descriptionHtml":"<p>Create a New Order for store</p>\n","status":"broken","statusMessage":"AttributeError: 'TestOrder' object has no attribute 'log'","statusTrace":"self = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_store_1___frugal_latido_com_np____Test_123____Ek__male___..._38____Silver____Black____No____40____Fit___8__7___33___11__5__9__12__46___6___5_6___Test_002___20000__1000___101011__>\nusername = 'frugal@latido.com.np', password = 'Test@123'\nproductname = 'Ek (male)', leatherProfile = 'Black Sheep', leatherSize = '38'\nhardware = 'Silver', lining = 'Black', polyfill = 'No', size = '40'\nbodytype = 'Fit', length = 8, chest = 7, waist = '33', hips = 11, shoulder = 5\nsleeves = 9, arms = 12, weight = 46, front = '6', armhole = 5.6\nremarks = 'Test 002', price = 20000, discount = 1000, OTP = '101011'\n\n    @allure.title(\"Create a New Order for store\")\n    @allure.description(\"Create a New Order for store\")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forStore\"))\n    @unpack\n    def test_create_order_store(self, username, password, productname, leatherProfile, leatherSize, hardware, lining,\n                                polyfill, size, bodytype, length, chest, waist, hips, shoulder, sleeves, arms, weight, front,\n                                armhole, remarks, price, discount, OTP):\n        # Login with Admin\n        self.LoginMethod.nativelogin(username, password)\n        try:\n>           self.assertEqual(\n                self.Order.createOrder_Store(productname, leatherProfile, leatherSize, hardware, lining, polyfill,\n                                             size, bodytype, length, chest, waist, hips, shoulder, sleeves, arms,\n                                             weight, front, armhole, remarks, price, discount, OTP),\n                \"Order Succesfully Created\",\n                msg=\"Verification Failed to Create a new user.\")\n\nTestOrder.py:93: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_store_1___frugal_latido_com_np____Test_123____Ek__male___..._38____Silver____Black____No____40____Fit___8__7___33___11__5__9__12__46___6___5_6___Test_002___20000__1000___101011__>\nfirst = None, second = 'Order Succesfully Created'\nmsg = 'Verification Failed to Create a new user.'\n\n    def _patched_equals(self, first, second, msg=None):\n        try:\n>           old(self, first, second, msg)\n\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_store_1___frugal_latido_com_np____Test_123____Ek__male___..._38____Silver____Black____No____40____Fit___8__7___33___11__5__9__12__46___6___5_6___Test_002___20000__1000___101011__>\nfirst = None, second = 'Order Succesfully Created'\nmsg = 'Verification Failed to Create a new user.'\n\n    def assertEqual(self, first, second, msg=None):\n        \"\"\"Fail if the two objects are unequal as determined by the '=='\n           operator.\n        \"\"\"\n        assertion_func = self._getAssertEqualityFunc(first, second)\n>       assertion_func(first, second, msg=msg)\n\nC:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python310\\lib\\unittest\\case.py:845: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_store_1___frugal_latido_com_np____Test_123____Ek__male___..._38____Silver____Black____No____40____Fit___8__7___33___11__5__9__12__46___6___5_6___Test_002___20000__1000___101011__>\nfirst = None, second = 'Order Succesfully Created'\nmsg = \"None != 'Order Succesfully Created' : Verification Failed to Create a new user.\"\n\n    def _baseAssertEqual(self, first, second, msg=None):\n        \"\"\"The default assertEqual implementation, not type specific.\"\"\"\n        if not first == second:\n            standardMsg = '%s != %s' % _common_shorten_repr(first, second)\n            msg = self._formatMessage(msg, standardMsg)\n>           raise self.failureException(msg)\nE           AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new user.\n\nC:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python310\\lib\\unittest\\case.py:838: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_store_1___frugal_latido_com_np____Test_123____Ek__male___..._38____Silver____Black____No____40____Fit___8__7___33___11__5__9__12__46___6___5_6___Test_002___20000__1000___101011__>\nusername = 'frugal@latido.com.np', password = 'Test@123'\nproductname = 'Ek (male)', leatherProfile = 'Black Sheep', leatherSize = '38'\nhardware = 'Silver', lining = 'Black', polyfill = 'No', size = '40'\nbodytype = 'Fit', length = 8, chest = 7, waist = '33', hips = 11, shoulder = 5\nsleeves = 9, arms = 12, weight = 46, front = '6', armhole = 5.6\nremarks = 'Test 002', price = 20000, discount = 1000, OTP = '101011'\n\n    @allure.title(\"Create a New Order for store\")\n    @allure.description(\"Create a New Order for store\")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forStore\"))\n    @unpack\n    def test_create_order_store(self, username, password, productname, leatherProfile, leatherSize, hardware, lining,\n                                polyfill, size, bodytype, length, chest, waist, hips, shoulder, sleeves, arms, weight, front,\n                                armhole, remarks, price, discount, OTP):\n        # Login with Admin\n        self.LoginMethod.nativelogin(username, password)\n        try:\n            self.assertEqual(\n                self.Order.createOrder_Store(productname, leatherProfile, leatherSize, hardware, lining, polyfill,\n                                             size, bodytype, length, chest, waist, hips, shoulder, sleeves, arms,\n                                             weight, front, armhole, remarks, price, discount, OTP),\n                \"Order Succesfully Created\",\n                msg=\"Verification Failed to Create a new user.\")\n    \n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException, AttributeError) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Creating order for Store.\",\n                          attachment_type=AttachmentType.PNG)\n>           self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\nE           AttributeError: 'TestOrder' object has no attribute 'log'\n\nTestOrder.py:104: AttributeError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestOrder","time":{"start":1687500628072,"stop":1687500628072,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"testStage":{"description":"Create a New Order for store","status":"broken","statusMessage":"AttributeError: 'TestOrder' object has no attribute 'log'","statusTrace":"self = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_store_1___frugal_latido_com_np____Test_123____Ek__male___..._38____Silver____Black____No____40____Fit___8__7___33___11__5__9__12__46___6___5_6___Test_002___20000__1000___101011__>\nusername = 'frugal@latido.com.np', password = 'Test@123'\nproductname = 'Ek (male)', leatherProfile = 'Black Sheep', leatherSize = '38'\nhardware = 'Silver', lining = 'Black', polyfill = 'No', size = '40'\nbodytype = 'Fit', length = 8, chest = 7, waist = '33', hips = 11, shoulder = 5\nsleeves = 9, arms = 12, weight = 46, front = '6', armhole = 5.6\nremarks = 'Test 002', price = 20000, discount = 1000, OTP = '101011'\n\n    @allure.title(\"Create a New Order for store\")\n    @allure.description(\"Create a New Order for store\")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forStore\"))\n    @unpack\n    def test_create_order_store(self, username, password, productname, leatherProfile, leatherSize, hardware, lining,\n                                polyfill, size, bodytype, length, chest, waist, hips, shoulder, sleeves, arms, weight, front,\n                                armhole, remarks, price, discount, OTP):\n        # Login with Admin\n        self.LoginMethod.nativelogin(username, password)\n        try:\n>           self.assertEqual(\n                self.Order.createOrder_Store(productname, leatherProfile, leatherSize, hardware, lining, polyfill,\n                                             size, bodytype, length, chest, waist, hips, shoulder, sleeves, arms,\n                                             weight, front, armhole, remarks, price, discount, OTP),\n                \"Order Succesfully Created\",\n                msg=\"Verification Failed to Create a new user.\")\n\nTestOrder.py:93: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_store_1___frugal_latido_com_np____Test_123____Ek__male___..._38____Silver____Black____No____40____Fit___8__7___33___11__5__9__12__46___6___5_6___Test_002___20000__1000___101011__>\nfirst = None, second = 'Order Succesfully Created'\nmsg = 'Verification Failed to Create a new user.'\n\n    def _patched_equals(self, first, second, msg=None):\n        try:\n>           old(self, first, second, msg)\n\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_store_1___frugal_latido_com_np____Test_123____Ek__male___..._38____Silver____Black____No____40____Fit___8__7___33___11__5__9__12__46___6___5_6___Test_002___20000__1000___101011__>\nfirst = None, second = 'Order Succesfully Created'\nmsg = 'Verification Failed to Create a new user.'\n\n    def assertEqual(self, first, second, msg=None):\n        \"\"\"Fail if the two objects are unequal as determined by the '=='\n           operator.\n        \"\"\"\n        assertion_func = self._getAssertEqualityFunc(first, second)\n>       assertion_func(first, second, msg=msg)\n\nC:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python310\\lib\\unittest\\case.py:845: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_store_1___frugal_latido_com_np____Test_123____Ek__male___..._38____Silver____Black____No____40____Fit___8__7___33___11__5__9__12__46___6___5_6___Test_002___20000__1000___101011__>\nfirst = None, second = 'Order Succesfully Created'\nmsg = \"None != 'Order Succesfully Created' : Verification Failed to Create a new user.\"\n\n    def _baseAssertEqual(self, first, second, msg=None):\n        \"\"\"The default assertEqual implementation, not type specific.\"\"\"\n        if not first == second:\n            standardMsg = '%s != %s' % _common_shorten_repr(first, second)\n            msg = self._formatMessage(msg, standardMsg)\n>           raise self.failureException(msg)\nE           AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new user.\n\nC:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python310\\lib\\unittest\\case.py:838: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_store_1___frugal_latido_com_np____Test_123____Ek__male___..._38____Silver____Black____No____40____Fit___8__7___33___11__5__9__12__46___6___5_6___Test_002___20000__1000___101011__>\nusername = 'frugal@latido.com.np', password = 'Test@123'\nproductname = 'Ek (male)', leatherProfile = 'Black Sheep', leatherSize = '38'\nhardware = 'Silver', lining = 'Black', polyfill = 'No', size = '40'\nbodytype = 'Fit', length = 8, chest = 7, waist = '33', hips = 11, shoulder = 5\nsleeves = 9, arms = 12, weight = 46, front = '6', armhole = 5.6\nremarks = 'Test 002', price = 20000, discount = 1000, OTP = '101011'\n\n    @allure.title(\"Create a New Order for store\")\n    @allure.description(\"Create a New Order for store\")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forStore\"))\n    @unpack\n    def test_create_order_store(self, username, password, productname, leatherProfile, leatherSize, hardware, lining,\n                                polyfill, size, bodytype, length, chest, waist, hips, shoulder, sleeves, arms, weight, front,\n                                armhole, remarks, price, discount, OTP):\n        # Login with Admin\n        self.LoginMethod.nativelogin(username, password)\n        try:\n            self.assertEqual(\n                self.Order.createOrder_Store(productname, leatherProfile, leatherSize, hardware, lining, polyfill,\n                                             size, bodytype, length, chest, waist, hips, shoulder, sleeves, arms,\n                                             weight, front, armhole, remarks, price, discount, OTP),\n                \"Order Succesfully Created\",\n                msg=\"Verification Failed to Create a new user.\")\n    \n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException, AttributeError) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Creating order for Store.\",\n                          attachment_type=AttachmentType.PNG)\n>           self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\nE           AttributeError: 'TestOrder' object has no attribute 'log'\n\nTestOrder.py:104: AttributeError","steps":[],"attachments":[{"uid":"d0eb1322950c4e14","name":"Creating order for Store.","source":"d0eb1322950c4e14.png","type":"image/png","size":105776},{"uid":"f1c6cdabeba6985b","name":"log","source":"f1c6cdabeba6985b.txt","type":"text/plain","size":2539}],"parameters":[],"attachmentsCount":2,"shouldDisplayMessage":true,"stepsCount":0,"hasContent":true},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestOrder::0","time":{"start":1687502716177,"stop":1687502716177,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"labels":[{"name":"feature","value":"Order List"},{"name":"severity","value":"normal"},{"name":"suite","value":"TestOrder"},{"name":"subSuite","value":"TestOrder"},{"name":"host","value":"0day"},{"name":"thread","value":"2692-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"TestOrder"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":8,"unknown":0,"total":10},"items":[{"uid":"7e9870aa65ec6ec","status":"passed","time":{"start":1686891841579,"stop":1686891888894,"duration":47315}},{"uid":"98a311d652010e66","status":"passed","time":{"start":1686288903989,"stop":1686288954091,"duration":50102}},{"uid":"fa4fbfa21e808270","status":"passed","time":{"start":1685710845763,"stop":1685710887853,"duration":42090}},{"uid":"86ae4f931ef8828b","status":"passed","time":{"start":1685345023694,"stop":1685345065062,"duration":41368}},{"uid":"ebf7e6587f58dbf7","status":"passed","time":{"start":1685108086254,"stop":1685108127707,"duration":41453}},{"uid":"ed97b9efdc76a95b","status":"broken","statusDetails":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[11]/div/div[2]/div/div[2]/div[2]/form/input\"}\n  (Session info: chrome=113.0.5672.127)\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x00FC8893+48451]\n\t(No symbol) [0x00F5B8A1]\n\t(No symbol) [0x00E65058]\n\t(No symbol) [0x00E90467]\n\t(No symbol) [0x00E9069B]\n\t(No symbol) [0x00EBDD92]\n\t(No symbol) [0x00EAA304]\n\t(No symbol) [0x00EBC482]\n\t(No symbol) [0x00EAA0B6]\n\t(No symbol) [0x00E87E08]\n\t(No symbol) [0x00E88F2D]\n\tGetHandleVerifier [0x01228E3A+2540266]\n\tGetHandleVerifier [0x01268959+2801161]\n\tGetHandleVerifier [0x0126295C+2776588]\n\tGetHandleVerifier [0x01052280+612144]\n\t(No symbol) [0x00F64F6C]\n\t(No symbol) [0x00F611D8]\n\t(No symbol) [0x00F612BB]\n\t(No symbol) [0x00F54857]\n\tBaseThreadInitThunk [0x75467D59+25]\n\tRtlInitializeExceptionChain [0x777AB74B+107]\n\tRtlClearBits [0x777AB6CF+191]","time":{"start":1685024247863,"stop":1685024272874,"duration":25011}},{"uid":"d154d26a7c05f202","status":"passed","time":{"start":1684416777238,"stop":1684416821387,"duration":44149}},{"uid":"d53c9660fc824677","status":"passed","time":{"start":1683887505152,"stop":1683887547375,"duration":42223}},{"uid":"9c41f4a50d0fbc38","status":"passed","time":{"start":1683187632434,"stop":1683187674479,"duration":42045}}]},"tags":[]},"source":"90f8f0a12e35c595.json","parameterValues":[]}